src/briefcase/platforms/linux/appimage.py

```diff
@@ -44,14 +44,20 @@ def project_path(self, app):
 
     def binary_name(self, app):
         safe_name = app.formal_name.replace(" ", "_")
-        return f"{safe_name}-{app.version}-{self.tools.host_arch}.AppImage"
+        arch = LinuxDeploy.arch(self.tools.host_os, self.tools.host_arch)
+        return f"{safe_name}-{app.version}-{arch}.AppImage"
 
     def binary_path(self, app):
         return self.bundle_path(app) / self.binary_name(app)
 
     def distribution_path(self, app):
         return self.dist_path / self.binary_name(app)
 
+    def verify_tools(self):
+        """Verify the AppImage LinuxDeploy tool and its plugins exist."""
+        super().verify_tools()
+        LinuxDeploy.verify(tools=self.tools)
```

Comment:
Verify LinuxDeploy before building an entire Docker image and then informing the user that LinuxDeploy can't run on ARM.

Reply:
I think this is something I've discovered in #1392; if you don't specify an explicit platform to use and the image only supports a platform that's different than the host platform, you get this warning.

Nonetheless, (and ignoring the completely corrupted `PATH` setting), I think this is https://github.com/AppImage/AppImageKit/issues/965 which suggests that building x86-64 AppImages on Apple Silicon may not currently be possible.

I haven't fully processed that issue....so, I look more at this later.